src/test/java/studio/blockops/vyom/core/utils/RLPTest.java

```diff
@@ -0,0 +1,49 @@
+package studio.blockops.vyom.core.utils;
+
+
+
+import org.bouncycastle.util.encoders.Hex;
+// Please delete this class after unit test class
+public class RLPTest {
+	public static void main(String[] args) {

```

Comment:
Follow proper unit test flow, as performed in eckey.
Unit tests are not performed using main() method.
No System logs in unit tests.
Tests will be in src/test/java source package.